### ISSUE TITLE
settings.php oddity

### ISSUE THREAD

**Issue by andig** [andig/moode#19]
In `settings.php` you keep updating the session- if the worker has accepted and finished the task or not, e.g. https://github.com/andig/moode/blob/master/settings.php#L62

Wouldn't it make more sense to:
- move session/db update back to daemon.php
- add `waitWorker()` to `settings.php` as the other config tasks are doing?

That should take care of session being updated with latest values after worker finishes.

In case of latency change you're even doing special check for session being locked:

https://github.com/andig/moode/blob/master/settings.php#L94

Is this necessary with the above changes?

**Comment by moodeaudio** [comment]
Yes, it would make perfect sense as long as user gets page back immediately with their updated input while worker php is completing the “job” and making the db update in the background.
